README.md

```diff
@@ -1,37 +1,38 @@
-# Privacy Policy for Topics
+Sure, below is your updated Privacy Policy including the new permissions:
```

Comment:
This seems like an llm response. Maybe forgot to remove it? 🤖

Reply:
SEP

Reply:
DONE